site/_docs/reference.md

```diff
@@ -2652,6 +2652,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | m | expr1 <=> expr2                                | Whether two values are equal, treating null values as the same, and it's similar to `IS NOT DISTINCT FROM`
 | * | ACOSH(numeric)                                 | Returns the inverse hyperbolic cosine of *numeric*
 | s | ARRAY(expr [, expr ]*)                         | Construct an array in Apache Spark
+| s | ARRAY_APPEND(array, element)                   | Appends an *element* to the end of the *array* and returns the result. If the *array* itself is null, the function will return null. If an *element* to add is null, the null *element* will be added to the end of the *array*
```

Comment:
1. `If the *array* itself is null,` `itself` maybe redundant.
2. `Type of *element* should be similar to type of the elements of the array`. 
3. `If an *element* that is null, the null *element* will be added to the end of the *array*`

Reply:
fixed